test/e2e/config/copy.bpmn.diagram.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { copySync } from 'fs-extra';
+
+function copyDirectoryContentIfNotExists(dirName: string): void {
+  copySync(`${__dirname}/../../fixtures/bpmn/${dirName}`, `${__dirname}/../../../dist/static/diagrams/`, { overwrite: false });
+}
+
+copyDirectoryContentIfNotExists('diagram');
+copyDirectoryContentIfNotExists('navigation');
+copyDirectoryContentIfNotExists('non-regression');
+copyDirectoryContentIfNotExists('performance');
```

Comment:
⚠️ we lost to subfolder structure which prevent name overlapping. That is why the PageTester previously took a folder name to select in which folder diagrams were picked.
I suggest we keep the folder structure to avoid any side effects in the future.
This will also make explicit in test where we pick the diagrams from: current implementation makes too much magic.

Reply:
As we don't manipulate the same concept in the different tests, we should not have the same name.
In my opinion, if we keep the hierarical as before, we introduce a complexity in the tests for nothing. The tests should focus on the tested feature, and not how to initialize the page.

Reply:
My main concern is about the magic this introduces in the tests: it is unclear where the source diagrams are located (in which folder).
What do you think @aibcmars?

Reply:
We decided to keep the implementation proposed here because it should be enough.
In diagram.rendering.test.ts, we will lookup all files in the `diagram` folder as we do for non-regression diagrams. This will make explicit where the files are taken from in the test source file

Reply:
done with 5b8b9ae